src/global_stats.c

```diff
@@ -162,3 +162,12 @@ void DialectsGlobalStats_AddToInfo(RedisModuleInfoCtx *ctx) {
     RedisModule_InfoAddFieldULongLong(ctx, field, GET_DIALECT(RSGlobalStats.totalStats.used_dialects, dialect));
   }
 }
+
+void IndexsGlobalStats_UpdateLogicallyDeleted(int toAdd) {
+    INCR_BY(RSGlobalStats.totalStats.logically_deleted, toAdd);
+}
+
+void IndexesGlobalStats_AddToInfo(RedisModuleInfoCtx *ctx) {
+    RedisModule_InfoAddSection(ctx, "indexes_statistics");
+    RedisModule_InfoAddFieldULongLong(ctx, "total_logically_deleted_docs", READ(RSGlobalStats.totalStats.logically_deleted));
```

Comment:
Don’t we have some section this metric could fit in? Are we planning to add more fields to this section?
Consider adding it to an existing section (maybe the GC section?)

Reply:
I guess we can move it to GC, WDYT @meiravgri ?

Reply:
We already have index statistics section called "index"
I'd put it there along with `number_of_indexes` and `active_*` statistics

Reply:
Move to this wonderful function the other fields of the section